policies/branch-protection-sdks.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: branch_protection_sdks
+description: Organization branch protection policy for Microsoft Graph SDKs
+resource: repository
+where:
+  - repository.name.equals("microsoft-graph-toolkit", StringComparison.InvariantCultureIgnoreCase)
+configuration:
+  branchProtectionRules:
+  - branchNamePattern: main
+    requiredApprovingReviewsCount:
+      min: 1
+    # Must have a CODEOWNER approve for the PR to be merged.
+    requireCodeOwnersReview: true
+    # Require status checks to pass before merging. TODO: this value should be true, we should work to support this.
+    # Used with the requiredStatusChecks setting to specify which checks must pass for the PR to be merged.
+    requiresStrictStatusChecks: false
```

Comment:
We want to find out whether if this is set to true, whether that clobbers the repo level checks.